pkgs/tools/networking/passh/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, stdenv }:
```

Comment:
I don't think it matters, but you can drop this `lib` import and use `stdenv.lib` instead.

```suggestion
{ fetchFromGitHub, stdenv }:
```

Reply:
I consider stdenv.lib to be an antipattern as lib and stdenv have nothing to do with each other.